compliance/collection/nodeinventorizer/caching_inventorizer_test.go

```diff
@@ -0,0 +1,140 @@
+package nodeinventorizer
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stretchr/testify/suite"
+)
+
+type mockSleeper struct {
+	receivedDuration time.Duration
+	callCount        int
+}
+
+func (ms *mockSleeper) mockWaitCallback(d time.Duration) {
+	ms.receivedDuration = d
+	ms.callCount++
+}
+
+type TestComplianceCachingSuite struct {
+	suite.Suite
+	cs      CachingScanner
```

Comment:
```suggestion
	cs      *CachingScanner
```

Reply:
This should be obsolete now, but thanks for the suggestion 😄